### PR TITLE
New package: pantalaimon-0.6.4

### PR DESCRIPTION
**Pull request by non-Jedi**
All other commits in this PR are dependencies or transitive dependencies of pantalaimon.

### CODE REVIEW

**Comment by Piraty** [comment]
good work!
can you please rebase this branch

**Comment by non-Jedi** [comment]
Rebased and added commits with necessary package updates due to version bounds of newer pantalaimon/matrix-nio.

File: srcpkgs/python3-unpaddedbase64/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-unpaddedbase64'
 pkgname=python3-unpaddedbase64
 version=1.1.0
-revision=2
+revision=3
```
Comment (Piraty) [inline srcpkgs/python3-unpaddedbase64/template:4]: what is this revbump for?

Comment (non-Jedi) [reply srcpkgs/python3-unpaddedbase64/template:4]: As noted by the commit, this package was never built for python 3.8. Rev 2 is built for python 3.6.

**Comment by non-Jedi** [comment]
I've fixed the `pycompile_module` lint warnings. I will note that `xlint` was clean when I submitted this originally; that warning must have been added since.

File: srcpkgs/python-logbook/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'python-logbook'
+pkgname=python-logbook
```
Comment (Chocimier) [inline srcpkgs/python-logbook/template:2]: Please make it python 3 only, we do not want new python 2 packages.

Comment (non-Jedi) [reply srcpkgs/python-logbook/template]: Good point.

**Comment by non-Jedi** [comment]
Thanks!

**Pull request merged**